rubrik_cdm/physical.py

```diff
@@ -38,20 +38,50 @@ def add_physical_host(self, hostname, timeout=60):
             dict -- The full API response for `POST /v1/host`.
         """
 
-        self.log('Searching the Rubrik cluster for the current hosts.')
-        current_hosts = self.get('v1', '/host', timeout=timeout)
+        if isinstance(hostname, list):
+            self.log('Searching the Rubrik cluster for the current hosts.')
+            current_hosts = self.get('v1', '/host', timeout=timeout)
 
-        for host in current_hosts['data']:
-            if host['hostname'] == hostname:
-                return "No change requird. The host '{}' is already connected to the Rubrik cluster.".format(
-                    hostname)
+            for host in current_hosts['data']:
+                for single_host in hostname:
+                    if host['hostname'] == single_host:
+                        hostname.remove(single_host)
+                        self.log("No change required. The host '{}' is already connected to the Rubrik cluster. Removing Host and continuing".format(
```

Comment:
Just noticed that these are `self.log()`. We should be returning this message instead. On line 51 as well

Reply:
If I do return will that no exit the remainder of the loop? We need to check all hosts during this and remove the hosts that area already configured in Rubrik

Reply:
Ahhh, got it. Didn't look at the code enough. Was thinking it was an end all be all impotence check.

Let's remove the `No change required` part of the log to avoid any confusion. Generally speaking we only show that syntax in return statements for the checks.

Reply:
Will do and submit a new commit.